tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -483,7 +485,8 @@ function configure_containerd() {
 
 function remove_artifacts() {
 	echo "deleting kata artifacts"
-	rm -rf ${dest_dir}/*
+	local host_install_dir="/host/${dest_dir}"
```

Comment:
Is there merit in having `host_install_dir` as a script wide variable, so it can be used in `install_artifacts` and `remove_artifacts` without re-defining locally?

Reply:
We use that in basically 2 places, installing and removing the artefacts.  I think we can have it globally defined without any issue.